.prettierrc

```diff
@@ -1,17 +1,8 @@
 {
-    "printWidth": 100,
-    "jsxBracketSameLine": false,
-    "singleQuote": true,
-    "tabWidth": 2,
-    "trailingComma": "es5",
-    "semi": false,
-    "plugins": ["prettier-plugin-prisma"],
```

Comment:
plugins is removed also, intentional?

Reply:
Yes, pretty sure we don't need it. Correct @jharrell ?

Reply:
correct